src/components/Envelope.vue

```diff
@@ -483,6 +475,30 @@ export default {
 				id: this.data.databaseId,
 			})
 		},
+		hasSelectAcl() {
+			if (!this.data.myAcls) {
+				return true
+			}
+			return this.data.myAcls.indexOf('r') !== -1
+		},
+		hasSeenAcl() {
+			if (!this.data.myAcls) {
+				return true
+			}
+			return this.data.myAcls.indexOf('s') !== -1
+		},
+		hasArchiveAcl() {
+			if (!this.data.myAcls) {
+				return true
+			}
+			return this.data.myAcls.indexOf('w') !== -1
+		},
+		hasDeleteAcl() {
+			if (!this.data.myAcls) {
+				return true
+			}
+			return this.data.myAcls.indexOf('t') !== -1
```

Comment:
I'm not entirely sure, but we might need to check for `e` as well. I'll look at the RFCs once again.

Reply:
@GretaD could you check that please?

Reply:
I just tested it and it turns out that wee need both rights, `t` and `e`.

Reply:
```suggestion
			return this.data.myAcls.indexOf('t') !== -1 && this.data.myAcls.indexOf('e') !== -1
```

Reply:
are you sure we need && here and not || ?

Reply:
Both are needed due to IMAP implementation details.

Ref https://datatracker.ietf.org/doc/html/rfc4314.html#section-2.1